lib/lightning/accounts/user_notifier.ex

```diff
@@ -155,26 +154,86 @@ defmodule Lightning.Accounts.UserNotifier do
     """)
   end
 
-  defp build_email_body(data, digest) do
-    digest_lookup = %{daily: "day", monthly: "month", weekly: "week"}
+  def build_digest_url(workflow, start_date, end_date) do
+    URI.new!(
+      "#{LightningWeb.Router.Helpers.url(LightningWeb.Endpoint)}/projects/#{workflow.project_id}/runs?"
+    )
+    |> URI.append_query(
+      URI.encode_query(%{
+        "filters[body]" => true,
+        "filters[crash]" => true,
+        "filters[failure]" => true,
+        "filters[log]" => true,
+        "filters[pending]" => true,
+        "filters[search_term]" => "",
+        "filters[success]" => true,
+        "filters[timeout]" => true,
+        "filters[wo_date_after]" => "",
+        "ilters[wo_date_before]" => "",
+        "filters[date_after]" => start_date,
+        "filters[date_before]" => end_date,
+        "filters[workflow_id]" => workflow.id
+      })
+    )
```

Comment:
We should be using verified routes here, we need to add `use LightningWeb, :html` to add support for the `~p` sigil. From there we can use it to also build the query string:

```
params = %{page: 1, direction: "asc"}
~p"/posts?#{params}"
```

See: https://hexdocs.pm/phoenix/Phoenix.VerifiedRoutes.html